client/platform/web-girder/api/viame.service.ts

```diff
@@ -13,36 +13,19 @@ interface ValidationResponse {
   message: string;
 }
 
-/**
- * listDatasets gets static metadata from girder.
- *
- * Important: data loaded in this way will not have had their imageData or videoUrl
- * populated. Outside of viewer, these shoudln't be necessary.
- */
-async function listDatasets({
-  limit = 50, offset = 0, sort = 'name',
-}): Promise<{
-  items: GirderMetadata[];
-  total: number;
-}> {
-  // Adjust for -1 (everything) which girder accepts as 0
-  const limitAdjusted = limit < 0 ? 0 : limit;
-  const { data, headers } = await girderRest.get<GirderModel[]>('viame/dataset', {
-    params: { limit: limitAdjusted, offset, sort },
-  });
-  const datasets: GirderMetadata[] = data.map((d) => ({
-    ...d,
-    ...(d.meta as GirderMetadataStatic),
-    id: d._id,
-    imageData: [],
-    videoUrl: '',
-  }));
-  return {
-    items: datasets,
-    total: parseInt(headers['girder-total-count'], 10),
-  };
+export interface BrandData {
+  brand: boolean;
+  logo: string;
```

Comment:
Not needed for this PR, but favicon is still VIAME?

Reply:
favicon seems like a reasonable addition.